tribe-ext-link-events-to-website-url/index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar: Link Event Titles to Event Website URLs
+ * Description:     An extension that makes event titles link to the events' Website URLs when present.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Link_Event_Titles_to_Event_Website_URLs
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+class Tribe__Extension__Link_Event_Titles_to_Event_Website_URLs extends Tribe__Extension {
```

Comment:
Could we also shorten the class name a bit? That's 57 characters wide. A lot of style guides try to keep line widths to 80characters or less. If we ever try to reference a function inside this, like 'Tribe__Extension__Link_Event_Titles_to_Event_Website_URLs::instance()->make_event_titles_link_to_event_website()` we are already well over 80 characters. The shorter the title the quicker devs can parse your code.